packages/react-native-bpk-component-calendar/README.md

```diff
@@ -51,25 +52,31 @@ import { StyleSheet, View } from 'react-native';
 import { spacingBase } from 'bpk-tokens/tokens/base.react.native';
 import BpkCalendar, { SELECTION_TYPES } from 'react-native-bpk-component-calendar';
 
-const styles = StyleSheet.create({
-  container: {
-    flex: 1,
-    justifyContent: 'center',
-    padding: spacingBase,
-  },
-});
-
-export default () => (
-  <View style={styles.container}>
-    <BpkCalendar
-      selectionType={SELECTION_TYPES.single}
-      selectedDates={[new Date()]}
-      onChangeSelectedDates={newDates => {
-        console.warn("Date selection changed");
-      }}
-    />
-  </View>
-);
+class App extends Component {
+  constructor(props) {
+    super(props);
+
+    this.state = { selectedDates: [] };
+  }
+
+  handleNewDates = newDates => {
+    this.setState({
+      selectedDates: newDates,
+    });
+  };
+
+  render() {
+    const { selectionType, onChangeSelectedDates, ...rest } = this.props;
+    return (
+      <BpkCalendar
+        locale={'en-gb'}
+        selectionType={SELECTION_TYPES.range}
```

Comment:
Can we show `minDate` and `maxDate` to drive home how to create the dates in UTC correctly?